store/tikv/coprocessor_cache_test.go

```diff
@@ -73,7 +73,7 @@ func (s *testCoprocessorSuite) TestBuildCacheKey(c *C) {
 }
 
 func (s *testCoprocessorSuite) TestDisable(c *C) {
-	cache, err := newCoprCache(&config.CoprocessorCache{Enabled: false})
+	cache, err := newCoprCache(&config.CoprocessorCache{Enable: false})
```

Comment:
Default value is false, no need to do value assignment.

Reply:
Thanks! Good catch.